File: ServerSpace/src/main/java/Models/PersonaNaturalMainModel.java
package Models;

import EntityLayer.PersonaNaturalEntity;
import com.fasterxml.jackson.annotation.JsonProperty;
import java.sql.Timestamp;

public class PersonaNaturalMainModel {

    public PersonaNaturalMainModel() {
        this.personaNaturalId = 0;
        this.tipoDocumento = "";
        this.numDocumento = "";
        this.nombres = "";
        this.apellidoPaterno = "";
        this.apellidoMaterno = "";
        this.fechaRegistro = null;
        this.codUsuario = "";
    }

    public PersonaNaturalMainModel(PersonaNaturalEntity Ent) {
        this.personaNaturalId = Ent.getPersonaNaturalId();
        this.tipoDocumento = Ent.gettipoDocumento();
        this.numDocumento = Ent.getNumDocumento();
        this.nombres = Ent.getNombres();
        this.apellidoPaterno = Ent.getApellidoPaterno();
        this.apellidoMaterno = Ent.getApellidoMaterno();
        this.fechaRegistro = Ent.getFechaRegistro();
        this.codUsuario = Ent.getCodUsuario();
    }

    @JsonProperty("PersonaNaturalId")
    private int personaNaturalId = 0;

    public int getPersonaNaturalId() {
        return personaNaturalId;
    }

    public void setPersonaNaturalId(int personaNaturalId) {
        this.personaNaturalId = personaNaturalId;
    }

    @JsonProperty("TipoDocumento")
    private String tipoDocumento = "";

    public String getTipoDocumento() {
        return tipoDocumento;
    }

    public void setTipoDocumento(String tipoDocumento) {
        this.tipoDocumento = tipoDocumento;
    }

    @JsonProperty("NumDocumento")
    private String numDocumento = "";

    public String getNumDocumento() {
        return numDocumento;
    }

    public void setNumDocumento(String numDocumento) {
        this.numDocumento = numDocumento;
    }

    @JsonProperty("Nombres")
    private String nombres = "";

    public String getNombres() {
        return nombres;
    }

    public void setNombres(String nombres) {
        this.nombres = nombres;
    }

    @JsonProperty("ApellidoPaterno")
    private String apellidoPaterno = "";

    public String getApellidoPaterno() {
        return apellidoPaterno;
    }

    public void setApellidoPaterno(String apellidoPaterno) {
        this.apellidoPaterno = apellidoPaterno;
    }

    @JsonProperty("ApellidoMaterno")
    private String apellidoMaterno = "";

    public String getApellidoMaterno() {
        return apellidoMaterno;
    }

    public void setApellidoMaterno(String apellidoMaterno) {
        this.apellidoMaterno = apellidoMaterno;
    }
    @JsonProperty("FechaRegistro")
    private Timestamp fechaRegistro = null;

    public Timestamp getFechaRegistro() {
        return fechaRegistro;
    }

    public void setFechaRegistro(Timestamp fechaRegistro) {
        this.fechaRegistro = fechaRegistro;
    }

    @JsonProperty("CodUsuario")
    private String codUsuario = "";

    public String getCodUsuario() {
        return codUsuario;
    }

    public void setCodUsuario(String codUsuario) {
        this.codUsuario = codUsuario;
    }

}

File: Space/Space.Server/Model/PersonaNatural/PersonaNaturalMainModel.cs
using Space.EntityLayer;
using System.Text.Json.Serialization;

namespace Space.Server
{
    public class PersonaNaturalMainModel
    {
        public PersonaNaturalMainModel()
        {
            this.PersonaNaturalId = 0;
            this.TipoDocumento = String.Empty;
            this.NumDocumento = String.Empty;
            this.Nombres = String.Empty;
            this.ApellidoPaterno = String.Empty;
            this.ApellidoMaterno = String.Empty;
            this.FechaRegistro = DateTime.Now;
            this.CodUsuario = String.Empty;
        }

        public PersonaNaturalMainModel(PersonaNaturalEntity Item)
        {
            this.PersonaNaturalId = Item.PersonaNaturalId;
            this.TipoDocumento = Item.TipoDocumento;
            this.NumDocumento = Item.NumDocumento;
            this.Nombres = Item.Nombres;
            this.ApellidoPaterno = Item.ApellidoPaterno;
            this.ApellidoMaterno = Item.ApellidoMaterno;
            this.FechaRegistro = Item.FechaRegistro;
            this.CodUsuario = Item.CodUsuario;
        }

        [JsonPropertyName("PersonaNaturalId")] public int PersonaNaturalId { get; set; }
        [JsonPropertyName("TipoDocumento")] public String TipoDocumento { get; set; }
        [JsonPropertyName("NumDocumento")] public String NumDocumento { get; set; }
        [JsonPropertyName("Nombres")] public String Nombres { get; set; }
        [JsonPropertyName("ApellidoPaterno")] public String ApellidoPaterno { get; set; }
        [JsonPropertyName("ApellidoMaterno")] public String ApellidoMaterno { get; set; }
        [JsonPropertyName("FechaRegistro")] public DateTime FechaRegistro { get; set; }
        [JsonPropertyName("CodUsuario")] public String CodUsuario { get; set; }
    }
}
